spec/requests/users_spec.rb

```diff
@@ -1,7 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe 'GET /users' do
+    before do
+      User.create(name: 'Test User')
+      get users_path
+    end
+
+    it 'returns a successful response' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the user index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes the list of users in the response body' do
+      expect(response.body).to include('List of Users')
+    end
+  end
+
+  describe 'GET /users/:id' do
```

Comment:
- [ ] Great job working on this milestone. You should test the index and `show` actions and that should be reflected in your `describe`. See below.
```
describe 'GET /show' do
  ...
end
```